web-console/src/views/workbench-view/run-panel/run-panel.tsx

```diff
@@ -540,6 +523,42 @@ export const RunPanel = React.memo(function RunPanel(props: RunPanelProps) {
                         }}
                       />
                     </MenuItem>
+
+                    <MenuBoolean
+                      icon={IconNames.ROCKET_SLANT}
+                      text="Approximate COUNT(DISTINCT)"
```

Comment:
I think `Approximate COUNT(DISTINCT)` is now in the menu twice (scroll down)

Reply:
aw